src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,66 @@
+package parser;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import shop.Cart;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class JsonParserTest {
+
+    private static final String EXPECTED_FILE_TEXT = "{\"cartName\":\"cart-to-write\",\"realItems\":[],\"virtualItems\":[],\"total\":0.0}";
+    private static Parser parser;
+
+    @BeforeAll
+    public static void setUp() {
+        parser = new JsonParser();
+    }
+
+    @Disabled
+    @Test
+    public void testWriteToFile() {
+        Cart cartToWrite = new Cart("cart-to-write");
+
+        parser.writeToFile(cartToWrite);
+        String actualFileText = null;
+        try {
```

Comment:
I think you can remove try/catch here and just add an exception to the method signature. If this fails, it means that something is wrong and the test has completed its task - it did not miss the bug